.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "node": true,
+        "mocha": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            4
+        ],
+        "quotes": [
+            "error",
+            "double"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ]
+    }
```

Comment:
I would also consider adding `curly` and set it to `all`. I've seen brace-free blocks go bad too many times!